tests/gha/test_mr_publish_results.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+import sys
+
+REPO_DIR = Path(__file__).parents[2]
```

Comment:
this seems fragile - can we do this in a more robust way?

Reply:
I agree this looks ugly. 

Since `.github/` is not a package (has no `__init__.py`), I am not sure how to best do an import. 

If we decided to move the tests into `.github/` (see other comment), a relative import might work. 

Happy to hear more ideas